lib/exportEntities.js

```diff
@@ -0,0 +1,8 @@
+const fse = require('fs-extra');
+const cst = require('../constants');
+
+const exportEntities = entities => fse.writeJson(cst.exportFile, entities, { spaces: 2 });
```

Comment:
could you use 'spaces: 4' ? That's the preferred format for Jhipster and most web apps as far as I know.

Reply:
Will do